src/control/Control.Layers.js

```diff
@@ -346,7 +350,18 @@ L.Control.Layers = L.Control.extend({
 			                 (layer.options.maxZoom !== undefined && zoom > layer.options.maxZoom);
 
 		}
+	},
+
+	_expand: function () {
+		// Backward compatibility, remove me in 1.1.

```

Comment:
As just seen IRL, but for the record: given it was private method, I think we are kind already to keep backward compatibility and there is not need to go the formal process. :)